src/client/Microsoft.Identity.Client/Region/RegionSource.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.;
+
+namespace Microsoft.Identity.Client.Region
+{
+    internal class RegionSource
```

Comment:
Please rename or add a comment to show that this is for telemetry purposes only.

Reply:
Added a comment.